### PR TITLE
feat: lower governance proposal requirements and require minimum participation

### PR DESCRIPTION
**Pull request by philipdesmedt**

### CODE REVIEW

File: clarity/contracts/arkadiko-governance-v2-1.clar [hunk 1]
```diff
@@ -245,14 +248,14 @@
         (contract-of stake-pool-diko)
         (unwrap-panic (contract-call? .arkadiko-dao get-qualified-name-by-name "stake-pool-diko"))
       )
-      (err ERR-NOT-AUTHORIZED)
+      (err ERR-WRONG-POOL)
```
Comment (nieldeckx) [inline clarity/contracts/arkadiko-governance-v2-1.clar:251]: Suggest to use the same error in 'propose' and 'vote-against' as well.

**Review by nieldeckx — COMMENTED** [review]
Looks good!

File: scripts/governance-add-proposal.js [hunk 2]
```diff
@@ -16,12 +16,12 @@ async function transact() {
       tx.uintCV(43100),
       tx.uintCV(900),
       tx.stringUtf8CV('Add STX/WELSH Pair'),
```
Comment (Scotty0448) [inline scripts/governance-add-proposal.js:18]: I notice that this still says 'Add STX/WELSH Pair'.

Comment (philipdesmedt) [reply scripts/governance-add-proposal.js]: Fixe

**Pull request merged**